pyspecdata/ipy.py

```diff
@@ -161,7 +161,9 @@ def _print_plain_override_for_nddata(arg,p,cycle):
         import IPython.display as d
         arg_copy = arg.copy()
         arg_copy.human_units()
-        if len(arg_copy.dimlabels) == 1:
+        if arg_copy.data.size < 2:
+            print(arg_copy)
+        elif len(arg_copy.dimlabels) == 1:
```

Comment:
so that in jupyter notebook we can print the nddata in units even if the data we have is a single value (with or without errors)